blender/arm/make.py

```diff
@@ -432,7 +431,7 @@ def compilation_server_done():
         log.error('Build failed, check console')
 
 def build_done():
-    print('Finished in ' + str( math.ceil((time.time() - profile_time) * 100) / 100) + 's')
+    print('Finished in {:0.3f}'.format(time.time() - profile_time) + 's')
```

Comment:
I think you can get rid of the `+ 's'` by moving it into the string (right after `{0:3f}`)